containers/local.env

```diff
@@ -8,6 +8,11 @@ FILES_DB_BUCKET=testbucket
 FILES_DB_ACCESS_KEY=minioadmin
 FILES_DB_SECRET_KEY=minioadmin
 
+MINUTES_BETWEEN_METRICS_REFRESH=5
+MILLIS_BETWEEN_USER_METRICS=50
+MILLIS_BETWEEN_GETTING_PUB_KEY_METRICS=5
+MILLIS_BETWEEN_GETTING_PUB_KEY_KEY_METRICS=5
```

Comment:
I think these could just be one variable: millis_between_redis_calls, and I think 100 for all of these is probably the right place for us to start.